README.md

```diff
@@ -62,4 +62,15 @@ Example `module.exports` `plugins` section of `docusaurus.config.js`.
       }
     ]
   ],
+```
+
+### .gitignore
+
+Add the following to to .gitignore
```

Comment:
```suggestion
Add the following to the .gitignore file in the `web-docs` folder. If one is not available, you may create a new one.
```
Clarify which .Gitignore file the contents belong in, otherwise the paths should be updated so they are based on the root .Gitignore path.


Also, I'm not sure what your thoughts are on versioning an example .Gitignore for easy download/access, but we could do something like
```suggestion
Add the following to the .gitignore file in the `web-docs` folder. If one is not available, you may create a new one or download the completed example: https://raw.githubusercontent.com/..../commondocs/main/...
```

Reply:
changed, but with different wording than suggested. 

I think the starter kit should have a web-docs/.gitignore, really that probably is where a "real" version should live.

Reply:
I'm fine with this. I was torn between suggesting this or just in the starter project anyways.